sdk/js/src/CacheStore.ts

```diff
@@ -24,17 +31,31 @@ export abstract class CacheStore {
         return this.store?.getItem(storeKey)
     }
 
+    private getConfigStoreKey(user: DVCPopulatedUser) {
+        return user.isAnonymous ? StoreKey.AnonymousConfig : StoreKey.IdentifiedConfig
+    }
+
+    private getUserIdStoreKey(user: DVCPopulatedUser) {
+        return user.isAnonymous ? StoreKey.AnonUserId : StoreKey.IdentifiedConfigUserId
+    }
+
+    private getConfigFetchDateKey(user: DVCPopulatedUser) {
+        return user.isAnonymous ? StoreKey.AnonymousConfigDate : StoreKey.IdentifiedConfigDate
+    }
+
     remove(storeKey: string): void {
         this.store?.removeItem(storeKey)
     }
 
-    saveConfig(data: BucketedUserConfig): void {
-        this.save(StoreKey.Config, data)
+    saveConfig(data: BucketedUserConfig, user: DVCPopulatedUser): void {
+        const configKey = this.getConfigStoreKey(user)
+        this.save(configKey, data)
         this.logger?.info('Successfully saved config to local storage')
     }
 
-    loadConfig(): BucketedUserConfig | null | undefined {
-        const config = this.load(StoreKey.Config)
+    loadConfig(user: DVCPopulatedUser): BucketedUserConfig | null | undefined {
+        const configKey = this.getConfigStoreKey(user)
+        const config = this.load(configKey)
```

Comment:
ideally this method could handle validating the ID and fetch date, and only attempt to parse the config if they're valid